packages/core/strapi/bin/strapi.js

```diff
@@ -311,19 +311,21 @@ program
       .choices(['exact', 'strict', 'subset', 'bypass'])
       .default('exact')
   )
+  .option('-d, --decompress', 'Decompress', false)
```

Comment:
We've never used booleans anywhere else in the Strapi CLI, we may prefer having --decompress and --no-decompress instead.

Reply:
I've removed those commands for the moment since we're going with auto-detection